packages/ui/src/models/visualization/flows/nodes/mappers/parallel-processor-base-node-mapper.test.ts

```diff
@@ -0,0 +1,96 @@
+import { createVisualizationNode } from '../../../visualization-node';
+import { ICamelElementLookupResult } from '../../support/camel-component-types';
+import { RootNodeMapper } from '../root-node-mapper';
+import { LoadBalanceNodeMapper } from './loadbalance-node-mapper';
+import { MulticastNodeMapper } from './multicast-node-mapper';
+
+describe('ParallelProcessorBaseNodeMapper', () => {
+  let mapper: MulticastNodeMapper | LoadBalanceNodeMapper;
+  let path: string;
+  let routeDefinition: unknown;
+
+  describe.each(['multicast', 'loadBalance'])("with '%s'", (processorName) => {
+    beforeEach(() => {
+      const rootNodeMapper = new RootNodeMapper();
+      rootNodeMapper.registerDefaultMapper(mapper);
+
+      if (processorName === 'multicast') mapper = new MulticastNodeMapper(rootNodeMapper);

```

Comment:
[side-comment] You could also pass each class alongside the name, like the following:

```
describe.each([
  ['multicast', MulticastNodeMapper],
  ['loadBalance', LoadBalanceNodeMapper]
])("with '%s'", (processorName, Mapper) => {
```

and then simply call the Mapper:
```
mapper = new Mapper(rootNodeMapper);
```